pkg/cluster/k8sres.go

```diff
@@ -969,6 +975,10 @@ func (c *Cluster) getPodEnvironmentConfigMapVariables() ([]v1.EnvVar, error) {
 func (c *Cluster) getPodEnvironmentSecretVariables() ([]v1.EnvVar, error) {
 	secretPodEnvVarsList := make([]v1.EnvVar, 0)
 
+	if len(c.Spec.Env) > 0 {
+		secretPodEnvVarsList = append(secretPodEnvVarsList, c.Spec.Env...)
+	}
```

Comment:
see comment above. Maybe we should variables from Secrets should take precedence over variables from config map.

Reply:
Fix logic, please review.